source/_components/device_tracker.unifi.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "Ubiquiti Unifi WAP"
+description: "Instructions how to use a Unifi WAP controller as a device tracker module."
+date: 2016-02-19 20:59
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ubiquiti.png
+ha_category: Presence Detection
```

Comment:
Oops you're right. 

The tags are not being used yet but I plan to use it to highlight the more useful components on the site